### PR TITLE
[README.md] Update with serveral parts

### PR DESCRIPTION
**Pull request by kygx-legend**
1. Add tarvis ci status.
2. Add version information about dependencies.
3. Add unit test information.

### CODE REVIEW

**Comment by kygx-legend** [comment]
@ddmbr @zzxx-husky @Yuzhen11 Please review!

**Comment by kygx-legend** [comment]
I found `github` has a new function named `squash and merge`, which claims that the commits in a PR would be squashed into one commit and then added to the base branch. Not sure if this works as I thought. We could try!

File: README.md [hunk 1]
```diff
@@ -9,22 +11,27 @@ Dependencies
...
 Husky has the following minimal dependencies:
 
-* CMake
-* ZeroMQ (libzmq and cppzmq)
-* Boost
-* A working C++ compiler (clang/gcc/icc/MSVC)
-* TCMalloc
+* CMake (Version >= 3.0.2)
+* ZeroMQ (including both [libzmq](https://github.com/zeromq/libzmq) and [cppzmq](https://github.com/zeromq/cppzmq))
+* Boost (Version >= 1.5.8)
```
Comment (ddmbr) [inline README.md]: Should be >= 1.58?

Comment (kygx-legend) [inline README.md]: Modified.

**Pull request merged**